src/core/codegen.rs

```diff
@@ -9,21 +9,21 @@ pub trait CodeGenerator {
 }
 
 pub fn generate(node: IRNode) {
-    use IRNode as IR;
+    use IRNode::*;
```

Comment:
only use `*` in this scope, so namespace conflict is not need to be concered.
the context is clear here, `IR::` looks redundant

Reply:
Thanks! But the IR is defined in another file so I would want to use wildcard importing only when the definition is in the same file.

Reply:
i have removed all the wildcard import